frontend/javascripts/admin/welcome_ui.js

```diff
@@ -0,0 +1,147 @@
+        };
+
+  return <a {...linkProps}>{content}</a>;
+};
+
+type WhatsNextHeaderProps = {
+  activeUser: APIUser,
+  onDismiss: () => void,
+};
+
+export const WhatsNextHeader = ({ activeUser, onDismiss }: WhatsNextHeaderProps) => (
+  <div>
+    <div
+      style={{
+        backgroundColor: "rgba(88, 88, 88, 0.4)",
+        backgroundImage: "linear-gradient(0deg, rgb(255 255 255) 0%, rgb(231 247 255) 70%)",
+        position: "relative",
+      }}
+    >
+      <div style={{ position: "absolute", right: 0, top: 0, margin: 8 }}>
+        <Tooltip title="Don't show this again" placement="left">
+          <Icon type="close" onClick={onDismiss} />
+        </Tooltip>
+      </div>
+      <div className="welcome-header-content">
+        <img className="wk-logo" src="/assets/images/oxalis.svg" alt="webKnossos Logo" />
+        <div className="text-and-button-container">
+          <h1
+            style={{
+              color: "#1F2937",
+              marginLeft: 56,
+              fontWeight: 200,
+              fontSize: 64,
+              lineHeight: "150%",
+              marginBottom: 0,
+            }}
+          >
+            Welcome to webKnossos!
+          </h1>
+          <div
+            style={{
+              margin: "0 20px 0px 60px",
+              fontWeight: 300,
+              fontSize: 20,
+              lineHeight: "150%",
+              color: "#6B7280",
+            }}
+          >
+            <p
+              style={{
+                fontSize: 20,
+                lineHeight: 1.5,
+                marginTop: 0,
+                marginBottom: 8,
+              }}
+            >
+              Congratulations on your new webKnossos account! To hit the ground running, we
+              recommend the following steps for you:
+            </p>
+            <div className="whats-next-actions-grid">
+              <WhatsNextAction
+                title="Open a Demo Dataset"
+                description="Have a look at a public dataset to experience webKnossos in action."
+                href={features().publicDemoDatasetUrl}
+                icon={<Icon type="play-circle" className="action-icon" />}
+              />
+
+              <WhatsNextAction
+                title="Import Your Own Data"
+                description="Directly upload your data as a zip file."
+                to="/datasets/upload"
+                icon={<Icon type="cloud-upload" className="action-icon" />}
+              />
+              <WhatsNextAction
+                title="Learn How To Create Annotations"
+                description="Watch a short video to see how data can be annotated with webKnossos."
+                icon={<Icon type="plus-circle" className="action-icon" />}
+                href="https://youtu.be/W-dosptovEU?t=52"
+              />
+              <WhatsNextAction
+                title="Invite Your Colleagues"
+                description="Send invites to your colleagues that they can join your organization."
```

Comment:
Find das etwas hackelig in der Formulierung. Evtl.
```suggestion
                description="Send invites to your colleagues and ask them to join your organization."
```

Reply:
Whoops, I intended to write it as "send invites [...] SO that they can ...", but the word "so" was missing. Your suggestion sounds fine, too, though. I'll take yours :+1: